docs/AuthRBAC.md

```diff
@@ -193,17 +199,27 @@ const authProvider = {
 };
 ```
 
-For every restricted resource, ra-rbac calls `authProvider.getPermissions()` to get the permissions.
+For every restricted resource, ra-rbac calls `authProvider.getPermissions()` to get the permissions. In practice, the permissions are usually returned upon login rather than in the `authProvider` code. The authProvider stores the permissions in memory or localStorage.
 
```

Comment:
??

Reply:
It wasn't clear I agree. By reading this the first time, I wondered if it was necessary to use `getPermissionsFromRoles` to use `ra-rbac`, because nowhere we said "if you want to work with roles, you can use `getPermissionsFromRoles`".

Reply:
In "Roles and permissions", I added the following:

> The RBAC system relies on *permissions* only. It's the `authProvider`'s responsibility to map roles to permissions. See the [`authProvider` Methods](#authprovider-methods) section for details.